### PR TITLE
Cards vue B-E additional

### PR DESCRIPTION
**Pull request by chosta**
All but two cards from A-to-E

### CODE REVIEW

File: src/cards/Cartel.ts [hunk 1]
```diff
@@ -16,4 +18,11 @@ export class Cartel implements IProjectCard {
       player.addProduction(Resources.MEGACREDITS, player.getTagCount(Tags.EARTH) + 1);
       return undefined;
     }
+    public metadata: CardMetadata = {
+      cardNumber: '137',
+      description: 'Increase your MC production 1 step for each Earth tag you have, including this',
```
Comment (kberg) [inline src/cards/Cartel.ts:23]: Missing `.` at end of the description.

File: src/cards/CommercialDistrict.ts [hunk 2]
```diff
@@ -48,4 +52,16 @@ export class CommercialDistrict implements IProjectCard {
...
     }
+    public metadata: CardMetadata = {
+      cardNumber: '085',
+      description: 'Decrease your energy production 1 step and increase your MC production 4 steps',
+      renderData: CardRenderer.builder((b) => {
+        b.productionBox((pb) => {
+          pb.minus().energy(1).br;
+          pb.plus().megacredits(4).br;
+        }).nbsp.nbsp.tile(TileType.COMMERCIAL_DISTRICT, true).br;
+        b.text('place this tile. 1 vp per adjacent city tile', CardRenderItemSize.TINY, true);
```
Comment (kberg) [inline src/cards/CommercialDistrict.ts:63]: Capital P for Place.

File: src/cards/render/CardRenderer.ts [hunk 3]
```diff
@@ -223,13 +223,13 @@ class Builder {
...
   }
 
-  public jovian(): Builder {
-    this._addRowItem(new CardRenderItem(CardRenderItemType.JOVIAN));
+  public earth(): Builder {
+    this._addRowItem(new CardRenderItem(CardRenderItemType.EARTH));
     return this;
   }
 
-  public venusTag(): Builder {
```
Comment (kberg) [inline src/cards/render/CardRenderer.ts:231]: Did you remove this altogether?

File: src/components/card/CardRenderTileComponent.ts [hunk 4]
```diff
@@ -13,14 +13,46 @@ export const CardRenderTileComponent = Vue.component('CardRenderTileComponent',
...
     getClasses: function(): string {
       const classes: string[] = ['card-tile'];
-      if (this.item.tile === TileType.BIOFERTILIZER_FACILITY) {
-        classes.push('card-special-tile--M');
-        classes.push('card-biofertilizer_facility--M');
+      const type: TileType = this.item.tile;
+      if (this.item.hasSymbol) {
+        classes.push('card-tile-canvass');
       }
+      if (type === TileType.BIOFERTILIZER_FACILITY) {
```
Comment (kberg) [inline src/components/card/CardRenderTileComponent.ts:20]: Keep in mind this is repeating code that displays tiles on the game board, so consider using that instead.

File: src/components/card/CardRenderTileComponent.ts [hunk 5]
```diff
@@ -13,14 +13,46 @@ export const CardRenderTileComponent = Vue.component('CardRenderTileComponent',
...
+        if (this.item.isAres) {
+          classes.push('card-tile-ares');
+          classes.push('board-space-tile--commercial_district_ares');
+        }
+        // normal commercial district uses symbol (see getHtml)
+      }
+
       return generateClassString(classes);
     },
+    // for tiles with symbols, we want to put symbol on top of tile canvass
```
Comment (kberg) [inline src/components/card/CardRenderTileComponent.ts:40]: Typo "canvass", also drop the use of "we" (grammar nit.) "Symbols for tiles go on top of the tile canvas."

File: src/cards/venusNext/ExtractorBalloons.ts [hunk 6]
```diff
@@ -59,7 +59,7 @@ export class ExtractorBalloons implements IActionCard, IProjectCard, IResourceCa
     description: 'Add 3 Floaters to this card',
     renderData: CardRenderer.builder((b) => {
       b.effectBox((eb) => eb.empty().startAction.floaters(1).description('Action: Add 1 Floater to this card')).br;
-      b.effectBox((be) => be.or(CardRenderItemSize.SMALL).floaters(2).startAction.venus(1).description('Action: Spend one floater here to draw 1 card')).br.floaters(3);
+      b.effectBox((be) => be.or(CardRenderItemSize.SMALL).floaters(2).startAction.venus(1).description('Action: add 1 Floater to this card, or remove 2 Floaters here to raise Venus 1 step')).br.floaters(3);
```
Comment (kberg) [inline src/cards/venusNext/ExtractorBalloons.ts]: Recommended reformat.

File: src/cards/turmoil/CulturalMetropolis.ts [hunk 7]
```diff
@@ -36,4 +39,15 @@ export class CulturalMetropolis implements IProjectCard {
       }
       return undefined;
     }
+    public metadata: CardMetadata = {
+      cardNumber: 'T03',
+      requirements: CardRequirements.builder((b) => b.party(PartyName.UNITY)),
+      description: 'Requires that Unity is ruling or that you have 2 delegates there. Decrease your energy production 1 step and increase your MC production 3 steps. Place a city tile. Place 2 delegates in 1 party',
```
Comment (kberg) [inline src/cards/turmoil/CulturalMetropolis.ts:45]: Missing period at the end of this sentence.

File: src/cards/CorporateStronghold.ts [hunk 8]
```diff
@@ -34,4 +35,15 @@ export class CorporateStronghold implements IProjectCard {
     public getVictoryPoints() {
       return -2;
     }
+    public metadata: CardMetadata = {
+      cardNumber: '182',
+      description: 'Decrease your Energy production 1 step and increase your MC production 3 steps. Place a City tile',
```
Comment (kberg) [inline src/cards/CorporateStronghold.ts:40]: Missing period.

File: src/cards/CupolaCity.ts [hunk 9]
```diff
@@ -31,4 +33,15 @@ export class CupolaCity implements IProjectCard {
         },
       );
     }
+    public metadata: CardMetadata = {
+      cardNumber: '029',
+      requirements: CardRequirements.builder((b) => b.oxygen(9).max()),
+      description: 'Oxygen must be 9% or less. Place a City tile. Decrease your Energy production 1 step and increase your MC production 3 steps',
```
Comment (kberg) [inline src/cards/CupolaCity.ts:39]: Period at end of sentence.

File: src/cards/ares/CapitalAres.ts [hunk 10]
```diff
@@ -2,8 +2,30 @@ import {IAdjacencyBonus} from '../../ares/IAdjacencyBonus';
...
+import {CardRenderItemSize} from '../render/CardRenderItemSize';
+import {CardRenderDynamicVictoryPoints} from '../render/CardRenderDynamicVictoryPoints';
 
 export class CapitalAres extends Capital {
   public name = CardName.CAPITAL_ARES;
   public adjacencyBonus: IAdjacencyBonus = {bonus: [SpaceBonus.MEGACREDITS, SpaceBonus.MEGACREDITS]};
+  public metadata: CardMetadata = {
+    cardNumber: 'A05',
+    description: {
+      text: 'Requires 4 ocean tiles. Place tile with ADJACENCY BONUS of 2MC. Energy prod -2 and MC prod +5',
```
Comment (kberg) [inline src/cards/ares/CapitalAres.ts:18]: +period.

File: src/cards/ares/CapitalAres.ts [hunk 11]
```diff
@@ -2,8 +2,30 @@ import {IAdjacencyBonus} from '../../ares/IAdjacencyBonus';
...
+      align: 'left',
+    },
+    requirements: CardRequirements.builder((b) => b.oceans(4)),
+    renderData: CardRenderer.builder((b) => {
+      b.productionBox((pb) => {
+        pb.minus().energy(2).br;
+        pb.plus().megacredits(5);
+      }).nbsp.tile(TileType.CAPITAL, false, true).br;
+      b.text('1 additional VP for each ocean tile adjacent to this city tile', CardRenderItemSize.TINY, true);
```
Comment (kberg) [inline src/cards/ares/CapitalAres.ts:27]: +period.

File: src/cards/ares/CommercialDistrictAres.ts [hunk 12]
```diff
@@ -2,8 +2,26 @@ import {IAdjacencyBonus} from '../../ares/IAdjacencyBonus';
...
+import {CardRenderItemSize} from '../render/CardRenderItemSize';
+import {CardRenderDynamicVictoryPoints} from '../render/CardRenderDynamicVictoryPoints';
+
 
 export class CommercialDistrictAres extends CommercialDistrict {
   public name = CardName.COMMERCIAL_DISTRICT_ARES;
   public adjacencyBonus: IAdjacencyBonus = {bonus: [SpaceBonus.MEGACREDITS, SpaceBonus.MEGACREDITS]};
+  public metadata: CardMetadata = {
+    cardNumber: 'A06',
+    description: 'Decrease your energy production 1 step and increase your MC production 4 steps',
```
Comment (kberg) [inline src/cards/ares/CommercialDistrictAres.ts:17]: +period.

File: src/cards/ares/CommercialDistrictAres.ts [hunk 13]
```diff
@@ -2,8 +2,26 @@ import {IAdjacencyBonus} from '../../ares/IAdjacencyBonus';
...
+    renderData: CardRenderer.builder((b) => {
+      b.productionBox((pb) => {
+        pb.minus().energy(1).br;
+        pb.plus().megacredits(4).br;
+      }).nbsp.nbsp.tile(TileType.COMMERCIAL_DISTRICT, false, true).br;
+      b.text('Place this tile which grants an adjacency bonus of 2MC. 1 vp per adjacent city', CardRenderItemSize.TINY, true);
```
Comment (kberg) [inline src/cards/ares/CommercialDistrictAres.ts:23]: +period.

**Review by kberg — APPROVED** [review]
Looks great!

File: src/cards/render/CardRenderer.ts (hunk 3 above)
Comment (chosta) [reply src/cards/render/CardRenderer.ts:231]: yes, it should be venus().played if we want a venus round tag (the blue one)

**Comment by chosta** [comment]
@kberg thanks for the review
I was intending to not use "." at all and add it to the description render function but that seems a bit odd now so I went and added all the missing "periods" from previous commits even.

**Pull request merged**